### ISSUE TITLE
CSS is not loading on v1.2.0

### ISSUE THREAD

**Issue by MUTOgen** [weirdpattern/gatsby-remark-embed-gist#36]
I see the same issue as here https://github.com/weirdpattern/gatsby-remark-embed-gist/issues/30 on the latest `1.2.0`
I've done an upgrade from `1.1.9` to use preload feature, but plugin looks broken for me now.

I use `gatsby-plugin-mdx` for MDX files. Here's a piece of config

```
{
      resolve: `gatsby-plugin-mdx`,
      options: {
        gatsbyRemarkPlugins: [
          {
            resolve: `gatsby-remark-images`,
            options: {
              maxWidth: 750,
              showCaptions: true,
            },
          },
          {
            resolve: "gatsby-remark-embed-gist",
            options: {
              includeDefaultCss: true,
              gistCssPreload: true,
            },
          },
          {
            resolve: `gatsby-remark-prismjs`,
            options: {
              classPrefix: "language-",
              inlineCodeMarker: null,
            },
          },
          "gatsby-remark-copy-linked-files",
          `gatsby-remark-autolink-headers`,
        ],
        extensions: [`.mdx`, `.md`],
      },
    },
```

How could I fix this?

**Comment by weirdpattern** [comment]
Can you try without the case preload?

**Comment by MUTOgen** [comment]
@weirdpattern The same problem

**Comment by weirdpattern** [comment]
Ok thanks for checking. I will have to take a closer look as some conflicts have been reported before when used with mdx..., In the meantime, if 1.1.9 was working for you, then I will suggest you rollback to that version

**Comment by MUTOgen** [comment]
@weirdpattern Ok, thanks!
Yea, I will stick to 1.1.9 for now

**Comment by weirdpattern** [comment]
One last thing, was this the only plug-in you updated?
Just wondering if there's another affecting the preload behavior

**Comment by MUTOgen** [comment]
@weirdpattern Actually it was mass update. Including `gatsby` itself.
I could share `package.json` and `gatsby-config` if it'll help

**Comment by weirdpattern** [comment]
Well, it can't hurt 😄 

I'm using version 1.2.0 with css preload on my own site... Everything is loading just fine... But I'm not using mdx... So maybe I will start with that...

Sorry for the inconveniences

**Comment by chitoku-k** [comment]
I have the same problem in v1.2.0 and I made a PR that potentially fixes this in #34 last month. How about trying with this fix?

**Comment by weirdpattern** [comment]
One easy way to find out, @MUTOgen could please try by adding the gistCssUrlAddress to your gatsby config?

```js
{
  resolve: "gatsby-remark-embed-gist",
  options: {
    includeDefaultCss: true,
    gistCssPreload: true,
    GistCssUrlAddress: "https://github.githubassets.com/assets/gist-embed-b3b573358bfc66d89e1e95dbf8319c09.css"
  },
},
```

**Comment by weirdpattern** [comment]
@chitoku-k I've merged your PR, and will push a new version to npm in a few hours, I think you are right and your PR will take care of this

**Comment by MUTOgen** [comment]
> One easy way to find out, @MUTOgen could please try by adding the gistCssUrlAddress to your gatsby config?
> 
> ```js
> {
>   resolve: "gatsby-remark-embed-gist",
>   options: {
>     includeDefaultCss: true,
>     gistCssPreload: true,
>     GistCssUrlAddress: "https://github.githubassets.com/assets/gist-embed-b3b573358bfc66d89e1e95dbf8319c09.css"
>   },
> },
> ```

Looks like this approach doesn't work for me.

**Comment by chitoku-k** [comment]
@MUTOgen How about setting `gistCssUrlAddress` rather than `GistCssUrlAddress`?

**Comment by weirdpattern** [comment]
Sorry, I was on my cellphone and the auto-caps kicked in... @chitoku-k is right, please try with the camelcased version

**Comment by weirdpattern** [comment]
Closed by #34 and #38 
If you still see the issue send me a note and we can reopen this.
